HW10/read.txt

```diff
@@ -0,0 +1,22 @@
+Java Script
+
+a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var i=a.length-3;
```

Comment:
you mean should be like this:
var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"]
 
 var i=a.length-1;

while ( i >= 0){
	console.log (a[i].toUpperCase());
	i=i-3;
 }